### PR TITLE
Installation script for interception and intercept.exe

### PR DESCRIPTION
**Pull request by jlndk**
### _Tl;dr: This PR adds some scripts that makes the tedious process of installing interception and intercept.exe much easier._

I've now completed the first draft to the script(s) discussed in issue #26.
Please say if something should be changed (like the way i write things) and I'll make sure to look at it :D. We should maybe also update the documentation, since this PR doesn't cover that.

As I discussed in the issue It was inpractical to have everything in one script, since interception requires a restart after installation anyways.  Because of that i setteled on two scripts: 'Installation.bat' and 'keymap.bat'.

## Installation.bat
This file downloads, extracts, and installs all the required programs for you. It automaticly tried to ask for administrator permissions, so that the user don't need to fiqure out how to open an elevated cmd.

## keymap.bat
After the installation is complete the user can use this file to automaticly generate a new keyremap.ini file that works with their keyboard. I've just copyied tarans full keyboard remap, but we could off cause change this template later if that's what we want.

### CODE REVIEW

**Comment by jlndk** [comment]
I actually just realised this. Should we also try to install AutoHotKey, if it's not currently on the system, or just assume that the user has it?

**Comment by TaranVH** [comment]
This all looks like very cool stuff. People are still able to access it, without me necessarily having to MERGE it, though, right?
I am not sure if i can or hsould MERGE, since I am very unfamiliar with how Github desktop works. Previously, it overwrote all of the stuff on my computer, undoing a lot of the changes that I had made.
and since I already have interception installed, I can't TEST this script to see exactly how it works, and i don't know if it really IS compatible with all the rest of my code.

it says that your Intercept/keyremap.ini file conflicts with my stuff. I am not yet sure how to resolve that. also you named it keyremap.org.ini ?

long story short, I never expected anyone to actually submit any pull requests at all, and I'm still not sure how to handle them. You've made a cool thing here, but I don't know how it works. So it might be easier to POINT people to it as an option, rather than do a full on MERGE.

**Comment by Griffdith** [comment]
On your fork of this (https://github.com/jlndk/2nd-keyboard/tree/master/Intercept), can you please give instructions of how to use your installer?
